conference.js

```diff
@@ -647,7 +652,11 @@ export default {
      * false.
      */
     isCallstatsEnabled () {
-        return room.isCallstatsEnabled();
+        if (room) {
```

Comment:
you can make this more compact by doing `return room && room.isCallstatsEnabled();`